### PR TITLE
Create subgraph index feedback part3

### PR DESCRIPTION
**Pull request by alch-msft**
1. Feedback from Charles to standardize terms
2. Additional details for resource quota trouble-shooting

### CODE REVIEW

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ ms.date: 2020-04-15
...
 
-## Transform MAG data to MAKES json entities
+## Transform MAG to MAKES entity files
 
-MAKES requires data it indexes to be placed in a single JSON file, with each line representing an individual entity. For this example, we will use a U-SQL script to filter and transform MAG data into JSON data which will then be used to create a custom index. To do this, you will need to go to your Azure Data Lake Analytics (ADLA) service instance and submit a new ALDA job to generate the text files containing academic data.
+To create MAKES index, you will need to format your data into MAKES entity files. MAKES entity file are text files, with each line containing a JSON string representing an individual entity. For this example, we will use a U-SQL script to filter and transform MAG data into JSON data which will then be used to create a custom index. To do this, you will need to go to your Azure Data Lake Analytics (ADLA) service instance and submit a new ALDA job to generate the text files containing academic data.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:20]: files

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 2]
```diff
@@ -149,14 +149,12 @@ You can check the build index job progress using Azure Management Portal. Each j
 >[!NOTE]
 > Preprocessor and indexer tasks are created in batches to avoid overwhelming the virtual machine workers. See jobManager task's output for more details.
 
-## Next steps
+## Deploy custom MAKES Index
 
-Now that the JSON data has been generated to build the custom index, we use the kesm tool to execute the build. You will follow the same procedures as deploying an index from your MAKES subscription, but replacing the **--MakesIndex** parameter with the path to the output of this example.
+Now that the custom MAKES index files has been generated. You can follow the same procedures as deploying an index from your MAKES subscription, but replacing the **--MakesIndex** parameter with the path to the output of this example.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:154]: have been generated.

**Comment by cyhuang01** [comment]
> ms.date: 2020-04-15

Update date.

---
Refers to: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:5 in 658c807. [](commit_id = 658c8074cba6176aead96829cd9ecdbe14e2546e, deletion_comment = False)

**Comment by cyhuang01** [comment]
> ms.date: 2020-04-15

Update date.

---
Refers to: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:5 in 658c807. [](commit_id = 658c8074cba6176aead96829cd9ecdbe14e2546e, deletion_comment = False)

**Pull request merged**